src/components/create/PegInAccountSelect.vue

```diff
@@ -141,17 +141,17 @@ export default class PegInAccountSelect extends Vue {
     if (this.pegInTxState.selectedAccount) {
       this.btcAccountTypeSelected = this.pegInTxState.selectedAccount;
     }
-
+    const isLiquality = this.pegInTxState.bitcoinWallet === constants.WALLET_LIQUALITY;
     this.accountBalances = [
       {
         text: this.getAccountBalanceText(constants.BITCOIN_SEGWIT_ADDRESS),
         value: constants.BITCOIN_SEGWIT_ADDRESS,
-        disabled: false,
+        disabled: isLiquality,
```

Comment:
I suggest changing this to a method that returns true or false according to certain conditions, and for now the only condition is that if the wallet is liquality it won't show segwit compatible nor legacy options

Reply:
I agree.